
Patch find_python.sh to find the python2.4 on older version of RHEL (3,4).

--- build/find_python.sh.old
+++ build/find_python.sh
@@ -5,7 +5,7 @@
 # Python 2.4 = 0x2040000
 VERSION=${1:-0x2040000}
 
-for pypath in "$PYTHON" "$PYTHON2" "$PYTHON3" python python2 python3; do
+for pypath in "$PYTHON" "$PYTHON2" "$PYTHON3" python2.4 python python2 python3; do
   if [ "x$pypath" != "x" ]; then
     DETECT_PYTHON="import sys;sys.exit((sys.hexversion < $VERSION) and 1 or 0)"
     if "$pypath" -c "$DETECT_PYTHON" >/dev/null 2>/dev/null; then
